website/src/pages/can-i-use/scenario-tiers.ts

```diff
@@ -0,0 +1,95 @@
+      "Azure_CLientGeneratorCore_ClientInitialization_*",
+      "Azure_Core_Basic_*",
+      "Azure_Core_Lro_*",
+      "Azure_Core_Model_*",
+      "Azure_Core_Page_list*",
+      "Azure_Core_Page_TwoModelsAsPageItem",
+      "Azure_Core_Scalar_*",
+      "Azure_Core_Traits_*",
+      "Azure_Encode_*",
+      "Azure_Example_*",
+      "Azure_Payload_*",
+      "Azure_Versioning_*",
+      "Client_ClientNamespace",
+      "Client_Naming_*",
+      "Client_Overload_*",
+      "Client_Structure_*",
+      "Resiliency_*",
+      // Azure Management Plane
+      "Azure_ResourceManager_CommonProperties_Error_*",
+      "Azure_ResourceManager_CommonProperties_ManagedIdentity_*",
+      "Azure_ResourceManager_LargeHeader_*",
+      "Azure_ResourceManager_NonResource_*",
+      "Azure_ResourceManager_OperationTemplates_*",
+      "Azure_ResourceManager_Resources_*",
+      // Specs
+      "Authentication_*",
+      "Encode_Bytes_*",
+      "Encode_Datetime_*",
+      "Encode_Numeric_*",
+      "Parameters_Basic_*",
+      "Parameters_BodyOptionality_*",
+      "Parameters_CollectionFormat_*",
+      "Parameters_Path_*",
+      "Parameters_Spread_*",
+      "Payload_ContentNegotiation_*",
+      "Payload_MediaType_*",
+      "Payload_MultiPart_*",
+      "Payload_Pageable_ServerDrivenPagination_link",
+      "Routes_fixed",
+      "Routes_InInterface",
+      "Routes_PathParameters_annotationOnly",
+      "Routes_PathParameters_explicit",
+      "Routes_PathParameters_templateOnly",
+      "Routes_QueryParameters_annotationOnly",
+      "Routes_QueryParameters_explicit",
+      "Routes_QueryParameters_QueryContinuation_Explode_primitive",
+      "Routes_QueryParameters_QueryContinuation_Standard_array",
+      "Routes_QueryParameters_QueryContinuation_Standard_record",
+      "Routes_QueryParameters_templateOnly",
+      "Serialization_EncodedName_Json_Property_get",
+      "Serialization_EncodedName_Json_Property_send",
+      "Server_*",
+      "SpecialHeaders_*",
+      "SpecialWords_*",
+      "Type_Array_*",
+      "Type_Dictionary_*",
+      "Type_Enum_*",
+      "Type_Model_*",
+      "Type_Property_*",
+      "Type_Scalar_*",
+      "Type_Union_EnumsOnly_get",
+      "Type_Union_EnumsOnly_send",
+      "Type_Union_FloatsOnly_get",
+      "Type_Union_FloatsOnly_send",
+      "Type_Union_IntsOnly_get",
+      "Type_Union_IntsOnly_send",
+      "Type_Union_MixedLiterals_get",
+      "Type_Union_MixedLiterals_send",
+      "Type_Union_MixedTypes_get",
+      "Type_Union_MixedTypes_send",
+      "Type_Union_ModelsOnly_get",
+      "Type_Union_ModelsOnly_send",
+      "Type_Union_StringAndArray_get",
+      "Type_Union_StringAndArray_send",
+      "Type_Union_StringExtensible_get",
+      "Type_Union_StringExtensible_send",
+      "Type_Union_StringExtensibleNamed_get",
+      "Type_Union_StringExtensibleNamed_send",
+      "Type_Union_StringsOnly_get",
+      "Type_Union_StringsOnly_send",
+      "Versioning_*",
+    ],
+    specialized: [],
```

Comment:
is there no specialized one right now? or an empty array means it picks the rest?